vmm/src/device_manager.rs

```diff
@@ -1975,6 +1984,30 @@ impl DeviceManager {
         Ok(Arc::new(Console { console_resizer }))
     }
 
+    fn add_tpm_device(
+        &mut self,
+        _interrupt_manager: &Arc<dyn InterruptManager<GroupConfig = LegacyIrqGroupConfig>>,
```

Comment:
Why are you passing a reference to the interrupt manager if you're not using it?